cli/cmd/internal/client/install.go

```diff
@@ -0,0 +1,87 @@
+package client
```

Comment:
I though `internal` meant "everything that comes after installing carrier" thus things you do on Carrier itself after it's installed. With this move now I don't understand the role of the `internal` directory.

Reply:
@viovanov Can you explain the `internal` directory to me ?
I am willing to move the `install` sources out of it if my move into it was wrong.

Reply:
`internal` is special in go. You can have public things without them being exported in an external package